### PR TITLE
[EX-4759] Update Cypress version

### PR DESCRIPTION
**Pull request by mavmaf**
# Pull request template
In order to update Cypress version, `npm install --save-dev cypress@8.4.0` was run and it was also needed to manually change the version in `packages/react-wrapper/package.json` and `packages/x-components/package.json`.

## Motivation and context
Several improvements have been made since v7.7.0. Some of them are the following:

- Projects will now open much faster on MacOS Big Sur with improved browser detection performance
- Fixed a regression in 7.2.0 that would cause cy.visit() to take longer to fire its load event in some circumstances
- Fixed a regression in 8.3.0 where the correct exit code would not be issued during cypress run-ct 
- Upgraded Chrome browser version used during cypress run 
- Multiple fixes for cy.intercept

## What is the destination branch of this PR?
- `Main`

## How has this been tested?

Run tests using Cypress

### CODE REVIEW

**Review by pmareke — APPROVED** [review]
Great job, everything works fine as before so for me it's ok!

**Review by pmareke — CHANGES_REQUESTED** [review]
In order to use the latest version of Cypress (8.4.1), please run `npm update`, discard the changes in the package.json and just leave the ones in the package-lock.json.

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "publish-release": "node scripts/publish-release"
   },
   "devDependencies": {
-    "conventional-changelog-conventionalcommits": "~4.6.0",
+    "conventional-changelog-conventionalcommits": "^4.6.1",
```
Comment (pmareke) [inline package.json:16]: Revert the changes of this file please, we only need the ones in the lock files.!

File: packages/react-wrapper/package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "@types/react": "~16.9.0",
     "@types/react-dom": "~16.9.0",
     "commitizen": "~4.1.2",
-    "cypress": "~7.7.0",
+    "cypress": "~8.4.1",
```
Comment (pmareke) [inline packages/react-wrapper/package.json:49]: Revert the changes of this file please, we only need the ones in the lock files.!

File: packages/x-components/package.json [hunk 3]
```diff
@@ -104,7 +104,7 @@
     "@vue/test-utils": "~1.0.3",
     "autoprefixer": "~9.7.4",
     "commitizen": "~4.0.3",
-    "cypress": "~7.7.0",
+    "cypress": "~8.4.1",
```
Comment (pmareke) [inline packages/x-components/package.json:107]: Revert the changes of this file please, we only need the ones in the lock files.!

**Review by pmareke — CHANGES_REQUESTED** [review]
Leave the changes only in the lock files.

**Review by CachedaCodes — DISMISSED** [review]

**Review by pmareke — APPROVED** [review]
Top!

**Review by CachedaCodes — APPROVED** [review]

**Pull request merged**